docs/content/reference/static-configuration/cli-ref.md

```diff
@@ -217,11 +217,29 @@ Timeout defines how long to wait on an idle session before releasing the related
 Local plugins configuration. (Default: ```false```)
 
 `--experimental.localplugins.<name>.modulename`:  
-plugin's module name.
+Plugin's module name.
+
+`--experimental.localplugins.<name>.settings`:  
+Plugin's settings (works only for wasm plugins).
+
+`--experimental.localplugins.<name>.settings.envs`:  
+Environment variables to forward to the wasm guest.
+
+`--experimental.localplugins.<name>.settings.mounts`:  
+Directory to mount to the wasm guest.
```

Comment:
I wonder if we should specify the syntax for this as IMHO it is very counter intuitive.